### PR TITLE
Convert html to text

### PR DESCRIPTION
**Pull request by Lexy2**
Hello @unsolaci,

I've built a PR based on your #103 and implemented in-memory conversion of HTML content to TXT, without first saving an HTML and then converting the file to TXT.

I tried different options, but your variant with HtmlAgilityPack provides the best results on my test set.

@tomchavakis, I also fixed a small typo and added pragmas to suppress GitHub warnings that we know about.

### CODE REVIEW

File: tests/NugetUtility.Tests/MethodsTests.cs [hunk 1]
```diff
@@ -224,7 +226,7 @@ public async Task GetPackages_AllowedLicenses_Should_Throw_On_MIT()
 
             result.Should().HaveCount(1);
             validationResult.IsValid.Should().BeFalse();
-            validationResult.InvalidPackages.Count.Should().Be(4);
+            validationResult.InvalidPackages.Count.Should().Be(5);
```
Comment (tomchavakis) [inline tests/NugetUtility.Tests/MethodsTests.cs:229]: Why the previous test fails now? I am not sure that I understand what changed for changing the counters?

File: tests/NugetUtility.Tests/MethodsTests.cs [hunk 2]
```diff
@@ -242,7 +244,7 @@ public async Task GetPackages_InputJson_Should_OnlyParseGivenProjects()
...
```
Comment (tomchavakis) [inline tests/NugetUtility.Tests/MethodsTests.cs:247]: same here

**Comment by tomchavakis** [comment]
@Lexy2 one more contribution. Thank you so much for your time and the quality of the PR's !!!
If you find some time a Test Case that covers this functionality should be perfect, otherwise, I'll add it later.

@unsolaci, please check if this PR works fine and covers your need.

File: tests/NugetUtility.Tests/MethodsTests.cs (hunk 1 above)
Comment (Lexy2) [reply tests/NugetUtility.Tests/MethodsTests.cs:229]: Yes. One more package in the project, one more package in the test.

File: tests/NugetUtility.Tests/MethodsTests.cs (hunk 2 above)
Comment (Lexy2) [reply tests/NugetUtility.Tests/MethodsTests.cs:247]: Yup, same thing here.

**Comment by Lexy2** [comment]
@tomchavakis thanks for your review! I'll leave the test case to you if you don't mind.

**Comment by tomchavakis** [comment]
@Lexy2, thank you again!!!

**Comment by unsolaci** [comment]
Hi @Lexy2!

I know that this is already merged, but I just wanted to thank you for
picking this up. I just tried it with my test set and it went well.

Thank you for all the improvements, this is awesome 🙏.

**Pull request merged**